distribution/changelog.txt

```diff
@@ -43,7 +43,7 @@
 - Fix: [#13266] Plugin API: Deleting key of sharedStorage not working.
 - Fix: [#13278] Desync caused by ghost tiles changing the ride mode.
 - Fix: [#13289] Litter and vomit sometimes not loading with RCT1 saved game or scenario
-- Fix: [#13292] Impossible excitement rating requirements with finish building 5 coasters goal
+- Fix: [#13334] Uninitialised variables in CustomTabDesc
```

Comment:
Looks like you made a mistake here. But a release is imminent, so it's better to wait with fixing and rebasing.